README.md

```diff
@@ -1,2 +1,19 @@
 # hw01
-Repository for homework 1
+# A Breif Bio for Sophia Mo
+I am a fourth-year undergraduate student at UChicago, majoring in Economics, Math, and Political Science. I am passionate about economics research, and plan to pursue graduate study in economics.
+
+## Background
+I was born in **Beijing, China**, and spent my whole life there before coming to the US for college.
+![Beijing](https://github.com/SophiaMo/hw01/blob/master/Image.png)
+
+## Interest
+I love reading, particularly fictions. Below are some of my favorite books:
+* [Housekeeping](https://www.goodreads.com/book/show/11741.Housekeeping); Marilynne Robinson
+* [Waterland](https://www.goodreads.com/book/show/148639.Waterland); Graham Swift
+* [Stoner](https://www.goodreads.com/book/show/166997.Stoner); John Williams
+* [History of the Rain](https://www.goodreads.com/book/show/19314428-history-of-the-rain); Niall Williams
+* [The Namesake](https://www.goodreads.com/book/show/33917.The_Namesake); Jhumpa Lahiri
+* [One True Thing](https://www.goodreads.com/book/show/176839.One_True_Thing); Anna Quindlen
+* [The Heart is a Lonely Hunter](https://www.goodreads.com/book/show/37380.The_Heart_Is_a_Lonely_Hunter); Carson McCullers
+* [Possession](https://www.goodreads.com/book/show/41219.Possession); A.S.Byatt
+* [Brideshead Revisited](https://www.goodreads.com/book/show/30933.Brideshead_Revisited); Evelyn Waugh
```

Comment:
Again, great job with the links! It would have been really easy to make a mistake in all these links and code but it all looks great!